Code-Hills/codehills-be
# Feature - User email notifications

# What does this PR do?
>- Create notifications 
>- Send email notifictaions

# Description of Task to be completed?
>- Create notifications model and migration
>- Configure email service
>- Send emails

## Requirements


- [x] Require dependencies installation
- [x] Require .env update

# How should this be manually tested?
> - Clone this repository
> - checkout to this branch `ft-user-email-notifications`
> - install the dependencies `npm install` and 
> - Run database migrations `npm run db:migrate` 
> - Go to `http://localhost:$PORT/api/v1/docs` in your browser to access the api documentation
>- For example deactivate an account and the user should receive an email

# Any background context you want to provide?
>- N/A
# What are the relevant pivotal tracker stories?
>- N/A
# Screenshots (if appropriate)

# Questions:
>- N/A
